app/waf/src/models/adc.model.ts

```diff
@@ -220,6 +220,20 @@ export class Adc extends CommonEntity {
   constructor(data?: Partial<Adc>) {
     super(data);
   }
+
+  getBasicAuth(): string {
+    return Buffer.from(
+      `${this.management.connection!.username}:${
+        this.management.connection!.password
```

Comment:
This username and password is used for storing the username/password for the openstack? If yes, when the timeout occurs, will these information be updated as well?

Reply:
They are username and password of bigip. Needn't to update them.